src/com/lftechnology/java/training/dipak/filesassignments/SearchACharacterInFile.java

```diff
@@ -0,0 +1,10 @@
+package com.lftechnology.java.training.dipak.filesassignments;
+
+
+public class SearchACharacterInFile {
+
+	public static void main(String[] args) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)